### PR TITLE
New subs showcase page 2

### PR DESCRIPTION
**Pull request by damien-shaw-guardian**
## Why are you doing this?
This is the new version of the subscription showcase page

[**Trello Card**](https://trello.com/c/OhnV0Nhw/2503-subscription-show-case-page-changes)

## Screenshots
support thegulocal com_uk_subscribe

### CODE REVIEW

File: support-frontend/assets/components/packshots/guardian-weekly-packshot.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <GridImage
+      gridId="subscriptionWeekly2"
+      srcSizes={[801, 477]}
+      sizes="(max-width: 740px) 100%,
+            (max-width: 980px) 175px,
+            250px"
+      imgType="png"
+    />
+    <GridImage
+      gridId="subscriptionWeekly3"
```
Comment (johnduffell) [inline support-frontend/assets/components/packshots/guardian-weekly-packshot.jsx]: why does the "3" one appear twice?

File: support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.jsx [hunk 2]
```diff
@@ -8,15 +8,17 @@ import { Provider } from 'react-redux';
 import Page from 'components/page/page';
 import FooterContainer from 'components/footer/footerContainer';
 import { detect, type CountryGroupId, AUDCountries, Canada, EURCountries, GBPCountries, International, NZDCountries, UnitedStates } from 'helpers/internationalisation/countryGroup';
-import SubscriptionsByCountryGroup from 'components/subscriptionsByCountryGroup/subscriptionsByCountryGroup';
+// import SubscriptionsByCountryGroup from 'components/subscriptionsByCountryGroup/subscriptionsByCountryGroup';
```
Comment (johnduffell) [inline support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.jsx]: assume this and the below is just a todo

File: support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.scss [hunk 3]
```diff
@@ -37,3 +37,535 @@
   }
 
 }
+
+[class^="subscriptions"] {
```
Comment (johnduffell) [inline support-frontend/assets/pages/subscriptions-landing/subscriptionsLanding.scss]: does this match any class starting with subscriptions?

File: support-frontend/assets/pages/subscriptions-landing/components/subscriptionsLandingContent.jsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+  const subsLinks = getSubsLinks(
+    countryGroupId,
+    referrerAcquisitionData.campaignCode,
+    getCampaign(referrerAcquisitionData),
+    referrerAcquisitionData,
+    abParticipations,
+    optimizeExperiments,
+  );
+  const abTest = null;
```
Comment (rupertbates) [inline support-frontend/assets/pages/subscriptions-landing/components/subscriptionsLandingContent.jsx]: Shouldn't this be assigned a value from somewhere? It looks like it's always null at the moment

**Pull request merged**